packages/core/src/telemetry/loggers.ts

```diff
@@ -422,6 +430,24 @@ export function logApiResponse(config: Config, event: ApiResponseEvent): void {
     model: event.model,
     type: 'tool',
   });
+
+  // Record OpenTelemetry GenAI semantic convention metrics
+  const { commonAttributes } = recordGenAiClientMetrics(config, event);
```

Comment:
`recordGenAIClientMetrics` is weirding me out a bit b/c if im understanding correctly, it seems to be doing two things at once: it's emitting an event about operation duration AND it's returning `commonAttributes` (which is a bit of confusing name since we already have `getCommonAttributes` in metrics.ts which is a different set of attributes)

I'm wondering about an alternative where `recordGenAiClientTokenUsage` and `recordGenAiClientOperationDuration` are always called explicitly, and maybe they can use a function to determine the "common" attributes dynamically each they they are called? it does mean (in this case) determining that twice instead of once but I'm not sure we need that optimization